kornia/geometry/subpix/spatial_soft_argmax.py

```diff
@@ -632,6 +633,8 @@ def conv_quad_interp3d(input: Tensor, strict_maxima_bonus: float = 10.0, eps: fl
         y_max += strict_maxima_bonus * new_nms_mask.to(input.dtype)
 
     dx_res: Tensor = dx.flip(1).reshape(B, CH, D, H, W, 3).permute(0, 1, 5, 2, 3, 4)
+    dx_res[:, :, [1, 2]] = dx_res[:, :, [2, 1]]
```

Comment:
```suggestion
    dx_res[:, :, (1, 2)] = dx_res[:, :, (2, 1)]
```
tuples are immutable

Reply:
done